recipes/intake-odbc/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "intake-odbc" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/intake-odbc-{{ version }}.tar.gz
+  sha256: 696c61ac767676bd46137907f7b57b08619a087d9ace2182f4ca400a88d9f473
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - intake
+    - turbodbc
+    - pyarrow
+    - pandas
+
+test:
+  imports:
+    - intake_odbc
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/intake/intake-odbc
+  summary: ODBC Plugin for Intake
+  license: BSD-2-Clause
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - martindurant
+    - blakerosenthal
```

Comment:
@rxm770 you are not adding yourself, correct?

Reply:
yes, @ocefpaf - confirmed not adding myself.